documentation/src/main/wiki/PerformanceTestsMethodology.md

```diff
@@ -0,0 +1,99 @@
+# Performance Tests Methodology
+
+## Introduction
+This page describes the methodology of Knot.x performance tests.
+
+## Goals
+The goals of the tests conducted are as follows:
+* Assess system performance.
+* Assess system stability.
+
+### Metrics
+During our tests, we focus on the following metrics which, when correlated, allow us to come up with number of observations about the system:
+* Throughput.
+* Response time - 90% Line.
+* Quantity of Virtual Users.
+* Number of errors (HTTP response codes > 400, Knot.x logs).
+* Infrastructure resources (cpu, memory, disk, network).
+* Application resources (jmx heap size, garbage collections).
+
+### KPIs
+* 1 second for response time (90% Line).
+* Error occurrences.
+* Stability:
+    * No memory leaks.
+    * No errors (logs, responses).
+    * Infrastructure resources OK.
+    * Application resources OK.
+	
+## Test Environment
+Test Environment is set up on AWS Cloud.
+![Test environment](assets/knotx-test-environment.png)
+
+### System under test
+* 1 x Apache - c4.large, 2CPU, 4GB RAM, 500Mbps bandwidth,centos 6.8.
+* 1 x Knot.x - c4.large, 2CPU, 4GB RAM, 500Mbps bandwidth,centos 6.8.
+* 1 x Mocks - c4.large, 2CPU, 4GB RAM, 500Mbps bandwidth,centos 6.8.
+* 1 x AEM - c4.xlarge, 4CPU, 8GB RAM, 750Mbps bandwidth.centos 6.8.
+
+### Knotx author start command
+`java -Dlogback.configurationFile=/content/knotx/author/logback.xml -Dvertx.cacheDirBase=/content/knotx/author/.vertx -Xms1024m -Xmx2048m -XX:ReservedCodeCacheSize=128m -Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.port=18092 -Dcom.sun.management.jmxremote.rmi.port=18092 -Djava.rmi.server.hostname=10.254.165.236 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false -Xloggc:/content/logs/knotx/gc.log -XX:+PrintGCApplicationStoppedTime -XX:+PrintGCApplicationConcurrentTime -XX:+PrintGC -XX:+PrintGCTimeStamps -XX:+PrintGCDetails -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=10 -XX:GCLogFileSize=5M -jar /content/knotx/author/knotx.jar -conf /content/knotx/author/config/author-config.json`
```

Comment:
It's really hard to see what's being set in those command, instead consider using code snippet as follows:
```
java -Dlogback.configurationFile=/content/knotx/author/logback.xml \
     -Dvertx.cacheDirBase=/content/knotx/author/.vertx \
     -Xms1024m -Xmx2048m -XX:ReservedCodeCacheSize=128m \
     -XX:ReservedCodeCacheSize=128m \
     -Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.port=18092 \
     ...
```
Additionally, I'd modify hostname IPs to something more generic (e.g. rmi.server.hostname)

This should be applied to remaining commands too